ArduCopter/mode_autorotate.cpp

```diff
@@ -57,6 +58,7 @@ bool ModeAutorotate::init(bool ignore_checks)
     _flags.straight_ahead_initial = 1;
     _flags.bail_out_initial = 1;
     _msg_flags.bad_rpm = true;
+	initial_energy_check =1;
```

Comment:
```suggestion
    initial_energy_check =1;
```

Reply:
Yeah I know.  I will run the style check on all of these files.